posters-overrides.json

```diff
@@ -4588,6 +4588,7 @@
   },
   {
    "number": 1621,
+   "title" : "[password for zoom room is visualqc] Site-differences and inter-rater reliability of visual QC for Freesurfer parcellations (VisualQC)",
```

Comment:
eh heh heh -- I would have recommended you to just use jitsi and not be special this way. Insofar you are the only one who requested override to zoom, and now you want to butcher your title to cater zoom needs? very suboptimal imho.  Can you reconsider or jitsi does not work for you? or do you hate privacy? ;-)

Reply:
NVM none of this matters, as I am unsubcribing from this poster sessions

Reply:
it is sad to see you leaving.  Meanwhile I will accept it anyways just in case you decide to participate again.  Also, as stated elsewhere, ability to run jitsi in parallel rooms is a super feature to help navigating and "manning" multiple posters at the same time.  so I would recommend to just remove this override or tune it to use jitsi